internal/config/config.go

```diff
@@ -0,0 +1,23 @@
+package config
+
+import (
+	"os"
+
+	"github.com/containers/common/pkg/config"
+	"github.com/sirupsen/logrus"
+)
+
+var containerConfig *config.Config
+
+func init() {
+	var err error
+	containerConfig, err = config.Default()
+	if err != nil {
+		logrus.Error(err)
+		os.Exit(1)
```

Comment:
I really think this is just a miss feature abusing init(). Of course it is a pre existing issue but this doesn't feel right at all. For starters exit code 1 is wrong as we document 125 as default exit code. But in general storing a copy of the config is just wrong.

So please just convert all the callers to `config.Default()` and handle the errors there. As @mtrmac pointed this here does not work with config.Reload()